app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -319,13 +346,14 @@ Future<Map<String, dynamic>?> _queryGraphQL(GraphQLClient client, int? prNumber)
 
 const String pullRequestChecksQuery = r'''
 query ChecksForPullRequest($sPullRequest: Int!) {
-  repository(owner: "flutter", name: "flutter") {
+  repository(owner: $sRepoOwner, name: $sRepoName) {
     pullRequest(number: $sPullRequest) {
       commits(last: 1) {
         nodes {
           commit {
             # (appId: 64368) == flutter-dashboard. We only care about
```

Comment:
@Piinks Do we need to change the `appId` for the engine?

Reply:
I do not think so, it is the same in `check_for_waiting_pull_requests_queries.dart`, which is used by the merge bot across multiple repositories. @godofredoc would know for sure though.